packages/http/src/webauthn-json/convert.ts

```diff
@@ -0,0 +1,80 @@
+// We export these values in order so that they can be used to deduplicate
+// schema definitions in minified JS code.
+
+import type { Schema, SchemaProperty } from "./schema-format";
+
+// TODO: Parcel isn't deduplicating these values.
+export const copyValue = "copy";
+export const convertValue = "convert";
```

Comment:
I'm not sure I understand what's going on here

Reply:
This is vendored code, so I'm not sure either! I'll move it to a vendor folder to make that explicit.